compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -1449,14 +1443,23 @@ pub enum BinOp {
     Add,
     /// Like `Add`, but with UB on overflow.  (Integers only.)
     AddUnchecked,
+    /// Like `Add`, but returns `(T, bool)` of both the wrapped result
+    /// and a bool indicating that it overflowed.
```

Comment:
```suggestion
    /// and a bool indicating whether it overflowed.
```
same for the other two